### DISCUSSION TITLE
How to load external subtitle file?

### DISCUSSION THREAD

**Discussion by Adithyan007hack** [Rafostar/clapper#234] [Q&A] [answered]
How to load an external subtitle file not embedded in the video

**Comment by Rafostar** [comment, marked as answer]
Currently by opening video alone (either from file manager or open files dialog) and then after playback starts, launching "Open Files" dialog again and opening subtitles file alone this time.

The original plan was to allow selecting and opening both either from file manager or open file dialog, but right now its broken, so until fixed subs needs to be opened after video starts. There is a issue filled for this bug: https://github.com/Rafostar/clapper/issues/231 if you want to track progress on fixing it.
